docs/pt/docs/tutorial/cookie-params.md

```diff
@@ -0,0 +1,33 @@
+# Parâmetros de Cookie
+
+Você pode definir parâmetros de Cookie da mesma maneira que define paramêtros com `Query` e `Path`.
+
+## Importe `Cookie`
+
+Primeiro importe `Cookie`:
+
+```Python hl_lines="3"
+{!../../../docs_src/cookie_params/tutorial001.py!}
+```
+
+## Declare parâmetros de `Cookie`
+
+Então declare os paramêtros de cookie usando a mesma estrutura que em `Path` e `Query`.
```

Comment:
```suggestion
Então declare os paramêtros de cookie usando a mesma estrutura que em `rota` e `consulta`.
```

Reply:
I explained in the topic above the guideline I followed to translate or not Path/Query mentions.